.editorconfig

```diff
@@ -0,0 +1,56 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = space
+tab_size = 4
```

Comment:
`tab_size = 4` → `indent_size = 4`

Reply:
I suggest to keep `tab_size` for the code that already use tabulations rather than spaces